### PR TITLE
Develop test fixes

### PR DESCRIPTION
**Pull request by cassandrakesewa**
Closes @un/react#

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

**Comment by justintemps** [comment]
Hey @cassandrakesewa, is this a WIP? I still see 44 / 103 test suites failing:

```
Test Suites: 44 failed, 103 passed, 147 total
Tests:       200 failed, 1009 passed, 1209 total
Snapshots:   1 failed, 3 written, 5 passed, 9 total
```

**Comment by cassandrakesewa** [comment]
> Hey @cassandrakesewa, is this a WIP? I still see 44 / 103 test suites failing:
> 
> ```
> Test Suites: 44 failed, 103 passed, 147 total
> Tests:       200 failed, 1009 passed, 1209 total
> Snapshots:   1 failed, 3 written, 5 passed, 9 total
> ```

@justintemps , Yes it's a WIP. I had to fix some few components that weren't rendering before I focus on the the tests

File: packages/react/src/components/Checkbox/Checkbox-test.js [hunk 1]
```diff
@@ -85,7 +83,7 @@ describe('Checkbox', () => {
     const label = wrapper.find('label');
     expect(label.length).toEqual(1);
     const span = wrapper.find('span');
-    expect(span.hasClass(`${prefix}--visually-hidden`)).toEqual(true);
+    expect(span.hasClass(`wfp--visually-hidden`)).toEqual(true);
```
Comment (Utzel-Butzel) [inline packages/react/src/components/Checkbox/Checkbox-test.js:86]: Can you please check why not use the `prefix` here?

File: packages/react/src/components/Footer/Footer.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import classnames from 'classnames';
 import Link from '../Link';
 import Wrapper from '../Wrapper';
 // import Icon from '../Icon';
-import { FooterExternal } from './FooterExternal';
+import {FooterExternal} from './FooterExternal';
```
Comment (Utzel-Butzel) [inline packages/react/src/components/Footer/Footer.js:7]: Would it be better to provide `<Footer />` and `<FooterExternal />` as two components?

File: packages/react/src/components/Search/Search-test.js [hunk 3]
```diff
@@ -97,13 +96,6 @@ describe('Search', () => {
         });
       });
 
-      describe('icons', () => {
```
Comment (Utzel-Butzel) [inline packages/react/src/components/Search/Search-test.js:100]: Why did you remove the tests here?

**Review by Utzel-Butzel — APPROVED** [review]
Thank you Cassandra! I made a few minor comments on the PR.

File: packages/react/src/components/Search/Search-test.js (hunk 3 above)
Comment (cassandrakesewa) [reply packages/react/src/components/Search/Search-test.js:100]: I removed the tests related to the Icon component because as we discussed last time, it will be deprecated and hence not worth have any test related to it. Let me know if you have any other thoughts :) @Utzel-Butzel

File: packages/react/src/components/Footer/Footer.js (hunk 2 above)
Comment (cassandrakesewa) [reply packages/react/src/components/Footer/Footer.js:7]: Sure, I can work on that

File: packages/react/src/components/Checkbox/Checkbox-test.js (hunk 1 above)
Comment (cassandrakesewa) [reply packages/react/src/components/Checkbox/Checkbox-test.js:86]: Previously, it was importing prefix from a wrong file which was causing it to throw an error. I wanted to point to the right file but later changed my mind because all the other test files are referencing `wfp`. I will have time to go back into all test files to use the prefix later. But for the mean time I just went with the usual format. @Utzel-Butzel

**Comment by justintemps** [comment]
Hey @cassandrakesewa and @Utzel-Butzel, I saw this got merged but it looks like a bunch of tests are still failing. Can I help with that? 

image

**Comment by cassandrakesewa** [comment]
@justintemps Robert was facing the same thing until he cleared a build file. I'm not sure the path of the file but it looks like there was some caching happening. Please check if you will find something like that, if not @Utzel-Butzel will come online around 2pm Rome time to help out.
